source/fundamentals/builders/aggregates.txt

```diff
@@ -562,3 +562,30 @@ and ``temperature`` fields:
    :end-before: // end setWindowFields
    :language: java
    :dedent:
+
+Atlas Search Metadata
+---------------------
+
+Use the ``searchMeta()`` method to create a
+:manual:`$searchMeta </reference/operator/aggregation/searchMeta/>`
+pipeline stage which returns only the metadata part of the results from
+Atlas full-text search queries.
+
+.. tip:: Only Available on Atlas for MongoDB v4.4.11 and later
+
+   This aggregation pipeline operator is only available
+   on :atlas:`MongoDB Atlas </>` clusters running v4.4.11 and later. For a
+   detailed list of version availability, see the MongoDB Atlas documentation 
+   on :atlas:`$searchMeta </atlas-search/query-syntax/#-searchmeta>`.
+
+The following example shows the ``count`` metadata for an Atlas search
+aggregation stage:
```

Comment:
issue: This doesn't match the format all the other builders are using, or your [other pr](https://github.com/mongodb/docs-java/pull/258/files#diff-bda0960da275f046f83f06fe4d7a6dc5e423e5b2c64c9cd1b8a08ebc85cd91ceR580-R581) - "The following example creates a pipeline stage that does something cool".

Should we keep with that or is it okay to differ here?

Reply:
This was an intentional decision.
The behavior of this operation is subsumed by the Atlas search aggregation stage, and it seemed more important to maintain the reader's focus on its behavior (retrieving the metadata for whatever specific Atlas search operation you decide to include instead of what the Atlas search operation that it describes is performing.
Also, since it will be adjacent to a description of the Atlas search builder, it seems redundant to repeat the description.

Let me know if your opinion differs.

Reply:
Nope, I just wanted to make sure we were making an intentional decision here.